### ISSUE TITLE
TERRAFORM CRASH on importing mongodbatlas_alert_configuration

### ISSUE THREAD

**Issue by cupuyc** [mongodb/terraform-provider-mongodbatlas#171]
Terraform import worked for other `mongodbatlas_alert_configuration`, but not for this one.

```
terraform import mongodbatlas_alert_configuration.replication_oplog_window 5e384f0dcf09a208528fb22d-5e384f0dcf09a208528fb22e
mongodbatlas_alert_configuration.replication_oplog_window: Importing from ID "5e384f0dcf09a208528fb22d-5e384f0dcf09a208528fb22e"...

Error: rpc error: code = Unavailable desc = transport is closing


panic: runtime error: invalid memory address or nil pointer dereference
```

[crash.log](https://github.com/terraform-providers/terraform-provider-mongodbatlas/files/4330913/crash.log)

**Comment by PacoDw** [comment]
Thank you so much for your review, let me check your log :)

**Comment by PacoDw** [comment]
@cupuyc I resolved it on #173 PR if you want to check it 👍 

Let us know if you have another comment or concern, thank you so much :)

**Comment by cupuyc** [comment]
Thanks :-)
